### ISSUE TITLE
Ask people to test vignettes

### ISSUE THREAD

**Issue by andrew-edwards** [pbs-assess/pacea#28]
I think the following have (or might!) agree to be guinea pigs for our vignettes, including Travis and Andy each testing each others. Please tick off a box when done. If you are unable to work through one, please just edit this issue and remove yourself from the list, or add a comment if that doesn't work, so we can keep track. Thanks!

- [ ] Travis and Andy to edit this to add details for each vignette (i.e. how to get started and build it).

ROMS output vignettes
- [ ] Andy
- [ ] Kelsey
- [ ] Andrea
- [ ] Brianna

Satellite SST and chlorophyll data when done:
- [ ] Andy
- [ ] Kelsey
- [ ] Andrea
- [ ] Brianna

Oceanic indices: 
- [ ] Travis
- [ ] Kelsey
- [ ] Andrea
- [ ] Brianna

SST buoy data: 
- [ ] Travis
- [ ] Kelsey
- [ ] Andrea
- [ ] Brianna

Recruitment/biomass output from stock assessment:
- [ ] Travis
- [ ] Kelsey
- [ ] Andrea
- [ ] Brianna

Add any more here.

**Comment by andrew-edwards** [comment]
My indices.Rmd one is ready for you to have a look over, @travistai2

**Comment by travistai2** [comment]
Comments for indices.Rmd vignette
by Travis Tai

- Provide other date format options from base R `as.Date`

as.Date("1999-01-01")
as.Date("01/01/99", format = "%d/%m/%y")	
as.Date("Mar 31, 2001", format = "%b %d, %Y")
?as.Date # for more date format options
	
- Plotting options for values vs anomalies
Help files for `plot.pacea_index`, value argument needs what types of values can be plotted ("value", "anomaly"...)

- Plotting style for indices.
I was a bit confused here. The default `plot(oni)` is a line plot with blue and red colours, but does the default change to a bar plot with plotting `smooth_over_year`, but the `style = "red_blue"` is the default? 
Playing around with it more, some indices default with a line plot, some with a bar plot. 
The `style` argument default has oni without a black line, but `red_blue` has the outline black line. What is the default argument (i.e. red blue without a black line)

**Comment by travistai2** [comment]
Comments for buoy_sst vignette
-by Travis Tai

Should there be an explanation as to why there are some gaps in the data? Perhaps in the Details of ?buoy_sst and the vignette.

I think the second paragraph of Details for `?buoy_sst` is a bit wordy which reads off as a bit confusing. Maybe "If you need data as up-to-date as possible then contact Andrew Edwards and he can update the package, or clone the developer package contents from GitHub." This is already assuming they know how to do so and the operations of using GitHub.

**Comment by travistai2** [comment]
I don't have any comments for the fish_populations vignette

**Comment by andrew-edwards** [comment]
> Comments for indices.Rmd vignette by Travis Tai
> 
> * Provide other date format options from base R `as.Date`
> 
> as.Date("1999-01-01") as.Date("01/01/99", format = "%d/%m/%y") as.Date("Mar 31, 2001", format = "%b %d, %Y") ?as.Date # for more date format options
> 

- [ ] I have the `lubridate-pacea-series()` function already (but it's only called from within a plotting function, and isn't fully general). I hadn't put full dates in pdo etc to emphasise they are monthly values (i.e. 2023-08-01 implies 1st August), though you have to pick a day anyway when plotting. So maybe I should add a `Date` column, for all time series, so that then we can have an `calc_anomaly()` function to calculate anomalies, that works on many series (e.g. hake recruitment, calculated bccm average temperature over a region, etc.). And have standardised plotting. Sound good? @travistai2

**Comment by briannamwright** [comment]
Comment for "Climatic and Oceanographic Indices: there appears to be no help file for the Southern Oscillation Index - typing in '?soi' elicits the message "No documentation for ‘soi’ in specified packages and libraries: you could try ‘??soi’"

**Comment by briannamwright** [comment]
Also for the Climatic Indices the default plotting labels for axes and tick marks are super tiny - I know that is all readily changeable in plotting options but maybe the default size could be a bit larger?
